dallinger/recruiters.py

```diff
@@ -591,12 +575,26 @@ def __init__(self):
         super(CLIRecruiter, self).__init__()
         self.config = get_config()
 
-    def exit_response(self, experiment, participant):
+    def assignment_submission_requested(self, experiment, participant):
         """Delegate to the experiment for possible values to show to the
         participant.
         """
         exit_info = sorted(experiment.exit_info_for(participant).items())
 
+        if participant.status in SUBMITTABLE_STATUSES:
+            # Execute synchonously
+            worker_function(
+                event_type="AssignmentSubmitted",
+                assignment_id=participant.assignment_id,
+                participant_id=participant.id,
+            )
+        else:
+            logger.info(
+                "Participant status: '{}', so skipping approval process".format(
```

Comment:
Do you think 'approval' is the right word? I thought it would be 'submission' (the participant is requesting to submit, but actually they've already submitted, so we're skipping submission).

Reply:
Ha! I spotted that yesterday afternoon and then forgot to fix it. 100% agree. Thanks for catching!

Reply:
Fixed